storage/operation/stats.go

```diff
@@ -0,0 +1,168 @@
+package operation
+
+import (
+	"fmt"
+	"math"
+	"sort"
+	"sync"
+
+	"github.com/rs/zerolog"
+
+	"github.com/onflow/flow-go/module/util"
+	"github.com/onflow/flow-go/storage"
+)
+
+// Stats holds statistics for a single prefix group.
+type Stats struct {
```

Comment:
Faye used a great library for stats in [checkpoint_collect_stats](https://github.com/onflow/flow-go/blob/54e27a5fd038d08afdc047a9d6b801c9084a705f/cmd/util/cmd/checkpoint-collect-stats/stats_utils.go#L24) if you are interested. It would also give you percentiles.

Reply:
That library is great. However, it requires to pass in a list of values, which could potentially big. The current Stats only stores a few values, which is much cheaper. IMO, the most important stats is count and total size, I'm not sure  if it’s worth the overhead of storing all values just to compute percentiles.